src/spryker/helm/app/templates/application/nginx/ingress.yaml

```diff
@@ -5,18 +5,18 @@ kind: Ingress
 metadata:
   creationTimestamp: null
   labels:
-    app.service: {{ .Values.prefix }}nginx
-  name: {{ .Values.prefix }}nginx
+    app.service: {{ .Values.resourcePrefix }}nginx
+  name: {{ .Values.resourcePrefix }}nginx
 spec:
   rules:
-  {{- range $key, $value := .Values.docker.services.nginx.environment }}
-  - host: {{ $value | quote }}
+  {{ range $key, $value := .Values.docker.services.nginx.environment }}
+  - host: "{{ $value }}"
```

Comment:
why? the original should work fine

Reply:
yes not sure where that came from

Reply:
| quote was new work I did in my PRs, the ingress.yaml was copied before it, so didn't get the improvements